developer-docs-site/docs/guides/sign-a-transaction.md

```diff
@@ -0,0 +1,300 @@
+---
+title: "Creating a Signed Transaction"
+slug: "creating-a-signed-transaction"
+sidebar_position: 1
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Creating a Signed Transaction
+
+All transactions executed on the Aptos Blockchain must be signed. This requirement is enforced by the chain for security reasons.
+
+You can use the [Aptos REST API](https://fullnode.devnet.aptoslabs.com/spec.html) for this purpose. The Aptos server will generate the signing message, the transaction signature and will submit the signed transaction to the Aptos Blockchain. Also see the tutorial [Your First Transaction](../tutorials/first-transaction.md).
+
+However, you may prefer instead that your client application, for example, a hardware security module (HSM), be responsible for generating the signed transaction. Before submitting transactions, a client must:
+
+- Hash the transactions into bytes, and
+- Sign the bytes with the account private key. See [Accounts][account] for how account and private key works.
+
+This guide will introduce the concepts behind constructing a transaction, generating the appropriate message to sign, and various methods for signing within Aptos.
+
+:::info
+Code examples below are provided in Typescript.
+:::
+
+## Overview
+
+Creating a transaction that is ready to be executed requires the following four steps:
+
+* Prepare the unsigned transaction, known as a `RawTransaction`.
+* Generate the appropriate salt for signing the transaction.
+* Sign this salt and the transaction to produce an `Authenticator`, and
+* Derive a complete and signed transaction, consisting of an  `Authenticator` and a `RawTransaction`.
+
+See the below diagram showing a high-level flow.
+
+![creating-signed-transaction.svg](/img/docs/creating-signed-transaction.svg)
+
+Unsigned transactions are known as `RawTransaction`s. They contain all the information about how to execute an operation on an account within Aptos. But they lack the appropriate authorization with a signature or `Authenticator`.
+
+In Aptos Blockchain, all the data is encoded as [BCS][bcs] (Binary Canonical Serialization). Aptos supports many different approaches to signing but defaults to a single signer using [Ed25519][Ed25519]. The `Authenticator` produced during the signing gives authorization to the Aptos Blockchain to execute the transaction on behalf of the account owner.
+
+## Key concepts
+
+### Raw transaction
+
+Raw transactions consist of the following fields:
+
+* **sender** Account address of the sender.
```

Comment:
this isn't typed the same way as the rest of the fields are

* **sender** (Address): Account address of the sender